### PR TITLE
#209 update with edits as submitted to LSP

### PR DESCRIPTION
**Pull request by amilan17**
#209

### CODE REVIEW

**Comment by amilan17** [comment]
@tomkralidis - please review the changes in clause 7 and ignore the other diffs: https://github.com/wmo-im/wcmp2/pull/210/files#diff-ec717b6546ff6063369d5b07d05f9d101cb950ef1646da0ae885050de6c2ae0f

File: standard/requirements/core/REQ_record_creation_date.adoc [hunk 1]
```diff
@@ -3,5 +3,6 @@
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/record_creation_date*
 ^|A |A WCMP record SHALL provide a single `+properties.created+` property.
-^|B |The `+properties.created+` property SHALL NOT be repeated or used to document change history.
+^|B |The `+properties.created+` property SHALL not be repeated or used to document change history.
```
Comment (amilan17) [inline standard/requirements/core/REQ_record_creation_date.adoc:6]:
```suggestion
^|B |The `+properties.created+` property SHALL NOT be repeated or used to document change history.
```

File: standard/sections/annex_d_codelists.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 === Dataset status
 
-The links below provide some controlled vocabularies in support of dataset status, and may be
+The links below provide some controlled vocabularies in support of dataset status, and MAY be
```
Comment (amilan17) [inline standard/sections/annex_d_codelists.adoc:8]:
```suggestion
The links below provide some controlled vocabularies in support of dataset status, and may be
```

File: standard/sections/clause_0_front_material.adoc [hunk 3]
```diff
@@ -8,8 +8,8 @@ This document defines the content, structure, and encoding for the WMO Core Meta
 WCMP documents provide descriptive information for discovery of WMO resources.  WMO resources include, but are not
 limited to, data (NWP models, observations, forecasts and warnings, etc.), services/APIs, and processes.
 
-WCMP documents shall be encoded in GeoJSON (RFC 7946 footnote:[https://datatracker.ietf.org/doc/html/rfc7946]) as
-defined in this specification and shall be made available as HTTP crawlable files or via API provisioning as defined
+WCMP documents SHALL be encoded in GeoJSON (RFC 7946 footnote:[https://datatracker.ietf.org/doc/html/rfc7946]) as
```
Comment (amilan17) [inline standard/sections/clause_0_front_material.adoc:11]:
```suggestion
WCMP documents shall be encoded in GeoJSON (RFC 7946 footnote:[https://datatracker.ietf.org/doc/html/rfc7946]) as
```

File: standard/sections/clause_0_front_material.adoc [hunk 4]
```diff
@@ -8,8 +8,8 @@ This document defines the content, structure, and encoding for the WMO Core Meta
...
+defined in this specification and SHALL be made available as HTTP crawlable files or via API provisioning as defined
```
Comment (amilan17) [inline standard/sections/clause_0_front_material.adoc:12]:
```suggestion
defined in this specification and shall be made available as HTTP crawlable files or via API provisioning as defined
```

File: standard/sections/clause_0_front_material.adoc [hunk 5]
```diff
@@ -24,6 +24,6 @@ wmo, wis, wis2, weather, climate, water, metadata, discovery, search
 
 [big]*iii.    Security Considerations*
 
-Based on the WMO Unified Data Policy for the International Exchange of Earth System Data (Resolution 1 (Cg-Ext(2021) footnote:[https://library.wmo.int/doc_num.php?explnum_id=11113#page=9], exchanged data are classified as `+core+` or `+recommended+`.  Core data is considered fully open and unrestricted with no security considerations.  Recommended data may have access control defined.
+Based on the WMO Unified Data Policy for the International Exchange of Earth System Data (Resolution 1 (Cg-Ext(2021) footnote:[https://library.wmo.int/doc_num.php?explnum_id=11113#page=9], exchanged data are classified as `+core+` or `+recommended+`.  Core data is considered fully open and unrestricted with no security considerations.  Recommended data MAY have access control defined.
```
Comment (amilan17) [inline standard/sections/clause_0_front_material.adoc:27]:
```suggestion
Based on the WMO Unified Data Policy for the International Exchange of Earth System Data (Resolution 1 (Cg-Ext(2021) footnote:[https://library.wmo.int/doc_num.php?explnum_id=11113#page=9], exchanged data are classified as `+core+` or `+recommended+`.  Core data is considered fully open and unrestricted with no security considerations.  Recommended data may have access control defined.
```

File: standard/sections/clause_1_scope.adoc [hunk 6]
```diff
@@ -5,12 +5,12 @@ as part of the WIS2 Global Discovery Catalogue (GDC).
 
 The WMO Core Metadata Profile (WCMP) standard defined herein is an extension of the International Standard _OGC API - Records - Part 1: Core_.
 
-WCMP discovery metadata records shall be encoded as GeoJSON as defined in _OGC API - Records - Part 1: Core_.
+WCMP discovery metadata records SHALL be encoded as GeoJSON as defined in _OGC API - Records - Part 1: Core_.
```
Comment (amilan17) [inline standard/sections/clause_1_scope.adoc:8]:
```suggestion
WCMP discovery metadata records shall be encoded as GeoJSON as defined in _OGC API - Records - Part 1: Core_.
```

File: standard/sections/clause_1_scope.adoc [hunk 7]
```diff
@@ -5,12 +5,12 @@ as part of the WIS2 Global Discovery Catalogue (GDC).
...
 
 The primary purpose of WCMP is to describe **datasets**.  While WCMP can describe any
 resource (datasets, APIs and data reduction services, processes, analytics, etc.), the key
 component for international data exchange is data.  Services and APIs operating on resources
-may be represented as part of their own metadata or associated through WCMP links.
+MAY be represented as part of their own metadata or associated through WCMP links.
```
Comment (amilan17) [inline standard/sections/clause_1_scope.adoc:13]:
```suggestion
may be represented as part of their own metadata or associated through WCMP links.
```

File: standard/sections/clause_2_conformance.adoc [hunk 8]
```diff
@@ -1,14 +1,14 @@
 == Conformance
 
-Conformance with this standard shall be checked using the tests specified in Annex A (normative) of this document.
+Conformance with this standard SHALL be checked using the tests specified in Annex A (normative) of this document.
```
Comment (amilan17) [inline standard/sections/clause_2_conformance.adoc:3]:
```suggestion
Conformance with this standard shall be checked using the tests specified in Annex A (normative) of this document.
```

File: standard/sections/clause_2_conformance.adoc [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
 
 _OGC API - Records_ (OARec) provides a record metadata model in support of resource discovery.  This standard is an extension of _OGC API - Records - Part 1: Core_.  Conformance to this standard requires demonstrated conformance to the applicable Conformance Classes of _OGC API - Records - Part 1: Core_.
 
 Authors of discovery metadata records published within the WIS2 Global Discovery Catalogue (GDC) are required to comply with the
-WMO Core Metadata Profile (WCMP).  WCMP discovery metadata shall therefore be compliant with OGC API - Records - Part 1:
+WMO Core Metadata Profile (WCMP).  WCMP discovery metadata SHALL therefore be compliant with OGC API - Records - Part 1:
```
Comment (amilan17) [inline standard/sections/clause_2_conformance.adoc:8]:
```suggestion
WMO Core Metadata Profile (WCMP).  WCMP discovery metadata shall therefore be compliant with OGC API - Records - Part 1:
```

File: standard/sections/clause_2_conformance.adoc [hunk 10]
```diff
@@ -1,14 +1,14 @@
...
 Core: Requirements Class: Record Core.
 
-WMO shall publish guidance material to assist authors of WCMP discovery metadata for maintaining
+WMO SHALL publish guidance material to assist authors of WCMP discovery metadata for maintaining
```
Comment (amilan17) [inline standard/sections/clause_2_conformance.adoc:11]:
```suggestion
WMO shall publish guidance material to assist authors of WCMP discovery metadata for maintaining
```

File: standard/sections/clause_8_gdc.adoc [hunk 11]
```diff
@@ -2,13 +2,13 @@
 
 In addition to documenting and describing WIS2 data holdings, WCMP discovery metadata will be
 searchable via the <<_the_wis2_global_discovery_catalogue>> (GDC).  As part of discovery and search
-workflow, a GDC may choose to apply filters to help narrow search results.  This is realized by
+workflow, a GDC MAY choose to apply filters to help narrow search results.  This is realized by
```
Comment (amilan17) [inline standard/sections/clause_8_gdc.adoc:5]:
```suggestion
workflow, a GDC may choose to apply filters to help narrow search results.  This is realized by
```

File: standard/sections/clause_8_gdc.adoc [hunk 12]
```diff
@@ -2,13 +2,13 @@
...
 the API of the GDC, as well as applying facets in the catalogue.
 
 === Faceting
 
 The API of the GDC, in addition to allowing for spatial/temporal/attribute queries and filters,
-may choose to apply facets to WCMP records.  Facets are effective when applied against controlled
+MAY choose to apply facets to WCMP records.  Facets are effective when applied against controlled
```
Comment (amilan17) [inline standard/sections/clause_8_gdc.adoc:11]:
```suggestion
may choose to apply facets to WCMP records.  Facets are effective when applied against controlled
```

File: standard/sections/clause_7_normative_text.adoc [hunk 13]
```diff
@@ -1,129 +1,130 @@
-== The WMO Core Metadata Profile Record
+== Appendix F: WMO Core Metadata Profile (Version 2)
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:1]:
```suggestion
== Appendix F: WMO Core Metadata Profile (Version 2)
```
```suggestion
== WMO Core Metadata Profile (Version 2)
```

File: standard/recommendations/core/REC_pids.adoc [hunk 14]
```diff
@@ -2,5 +2,6 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Recommendation {counter:rec-id}* |*/rec/core/pids*
-^|A |A WCMP record SHOULD provide persistent identifier references via items in the `+properties.externalIds+` array property, where the value of `+scheme+` is based on an established persistent identifier scheme (such as `+https://doi.org+`, `+https://arks.org+`, `+https://handle.net+`, etc.), and the `+value+` property is the persistent identifier (for example, `https://dx.doi.org/10.14287/10000001`).
+^|A |A WCMP record SHOULD provide persistent identifier references via items in the `+properties.externalIds+` array property, where the value of `+scheme+` is based on an established persistent identifier scheme (such as `+https://doi.org+`, `+https://arks.org+` or `+https://handle.net+`), and the `+value+` property is the persistent identifier (for example, `https://dx.doi.org/10.14287/10000001`).
 |===
+//rec9
```
Comment (amilan17) [inline standard/recommendations/core/REC_pids.adoc:7]:
```suggestion
//rec10
```

File: standard/sections/clause_7_normative_text.adoc [hunk 15]
```diff
@@ -788,61 +792,61 @@ include::../requirements/core/REQ_data_policy.adoc[]
...
+|**Required**
+|The link destination or target, or URL.
 
 |``rel``
-|optional
+|Optional
 |The relationship that the link bears in the context of the WCMP record.  This is a controlled vocabulary defined
- by IANA link relationsfootnote:[https://www.iana.org/assignments/link-relations/link-relations.xhtml] or WCMP
- defined extensionsfootnote:[https://codes.wmo.int/wis/link-type] (see <<_link_relation_selection>>)
+ by link:https://www.iana.org/assignments/link-relations/link-relations.xhtml[IANA link relations] or link:https://codes.wmo.int/wis/link-type[WIS link type] extensions. (see table <<_link_relation_selection>>)
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:817]:
```suggestion
 by link:https://www.iana.org/assignments/link-relations/link-relations.xhtml[IANA link relations] or link:https://codes.wmo.int/wis/link-type[WIS link type] extensions. (see table <<_1_19_5_link_relation_selection>>)
```

File: standard/sections/clause_8_gdc.adoc
Comment (tomkralidis) [inline standard/sections/clause_8_gdc.adoc:1]: Links in this file need to be updated as follows:

```diff
diff --git a/standard/sections/clause_8_gdc.adoc b/standard/sections/clause_8_gdc.adoc
index 7ad9cda..a0ddda8 100644
--- a/standard/sections/clause_8_gdc.adoc
+++ b/standard/sections/clause_8_gdc.adoc
@@ -13,5 +13,5 @@ may choose to apply facets to WCMP records.  Facets are effective when applied a
 vocabularies and classifications.  The following WCMP required properties would be subject to faceting by a GDC:
 
 - ``centre-id`` as defined in <<_identifier>>
-- <<_properties_type>>
-- <<_properties_wmo_data_policy>>
+- <<_1_6_properties_type>>
+- <<_1_18_properties_wmo_data_policy>>
```

**Review by tomkralidis — CHANGES_REQUESTED** [review]
See change requests. As well, need to review/assess #214/#215 in relation to this PR.

File: standard/sections/clause_7_normative_text.adoc [hunk 16]
```diff
@@ -1,129 +1,130 @@
-== The WMO Core Metadata Profile Record
+== WMO Core Metadata Profile (Version 2)
+Note: This section of working draft document is the same as Appendix F in the https://library.wmo.int/idurl/4/68731[_Manual on the WMO Information System_] (WMO-No. 1060), Volume II
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:2]:
```suggestion
Note: This section of working draft document is the same as Appendix F in the https://library.wmo.int/idurl/4/68731[_Manual on the WMO Information System_] (WMO-No. 1060), Volume II.
```

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**